dockers/screenshotter/screenshotter.js

```diff
@@ -216,67 +234,49 @@ function startServer() {
     }
     const config = {
         ...webpackConfig.devServer,
-        port,
+        port: katexPort,
         hot: false,
         liveReload: false,
         client: false,
     };
     const compiler = webpack(webpackConfig);
-    const wds = new WebpackDevServer(config, compiler);
-    wds.start(port).then(() => {
-        devServer = wds;
-        katexPort = port;
-        attempts = 0;
-        process.nextTick(opts.seleniumProxy ? getProxyDriver
-            : opts.browserstack ? startBrowserstackLocal : tryConnect);
-    })
-    .catch((err) => {
-        if (devServer !== null) { // error after we started listening
-            throw err;
-        } else if (++attempts > 50) {
-            throw new Error("Failed to start up dev server");
-        } else {
-            process.nextTick(startServer);
-        }
-    });
+    devServer = new WebpackDevServer(config, compiler);
```

Comment:
When testing locally, I noticed that this enables Webpack's `watch` feature (partly because it gives me some warnings when it runs), which we don't need for screenshotting. Not directly related to this PR, but perhaps it's worth turning that off?